rust/sealevel/programs/mailbox/src/processor.rs

```diff
@@ -76,6 +79,16 @@ pub fn process_instruction(
     })
 }
 
+struct Foo(u32);
```

Comment:
I added this to test something before lol - can you remove all changes from this file